pandas/tests/indexes/period/test_ops.py

```diff
@@ -333,3 +334,16 @@ def test_freq_setter_deprecated(self):
         # warning for setter
         with pytest.raises(AttributeError, match="can't set attribute"):
             idx.freq = pd.offsets.Day()
+
+
+@pytest.mark.xfail(reason="PeriodIndex.sort_values currently unstable")
```

Comment:
well its doesn't work :->, do we have an issue tracking this, pls add the refernce in the xfail (and if we don't pls create an issue)

Reply:
Couldn't find an existing issue, so I opened #35922 35922

Reply:
kk add the issue number to the xfail message

Reply:
though I think the comprehensive test below covers this? (not averse to having this as well, just asking)